MapboxCoreNavigation/RouteController.swift

```diff
@@ -194,6 +193,20 @@ open class RouteController: NSObject, Router {
         locationManager.stopUpdatingHeading()
         locationManager.delegate = nil
     }
+    
+    /**
+     End the current navigation session.
+     */
+    @objc public func endNavigation(rating: Int = MMEEventsManager.unrated, comment: String? = nil) {
```

Comment:
Considering all the logic inside of deinit was already private, what do you think about keeping this function private?